CHANGELOG.md

```diff
@@ -4,6 +4,22 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
 
 ## 0.17.0 (2024/07/17)
 
+### New and Improved
+
+* SBC (Storage Bucket Credential): This release introduces, SBC, a resource that represents credentials for 
+authentication and authorization with an external object store. There are two SBC types, managed secret and environmental.
+([PR]((https://github.com/hashicorp/boundary/pull/4933))), ([PR]((https://github.com/hashicorp/boundary-plugin-minio/pull/18))) and ([PR]((https://github.com/hashicorp/boundary-plugin-aws/pull/46)))
```

Comment:
I think these PR links might have messed up formatting, it looks like there's an extra layer of parentheses

Reply:
great catch, fixed!